### DISCUSSION TITLE
[3주차][온라인 스터디] 글 모음

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/fe-interview-basecamp#178] [General]
**순서**

~22:05 체크인. 자신의 PR 및 주제 작성

~22:20 다른 PR 참조하여 내 PR 수정하기. 궁금한 점, 모르는 점은 PR에 질문달기

~22:35 입풀기(with 카훗)

~23:00 3인 모의면접. 1:1로 진행하며, 남은 1명은 관찰자.

**Comment by bigsaigon333** [comment]
## 체크인

현재의 기분을 10점 만점으로 표현해주세요~ 이유도 적어주세요!

**Reply by bigsaigon333** [reply]
6점 제가 지금 이력서 스터디를 하고 있는데.. 아직 끝이 안났네요 :dizzy_face: 
카훗 하기 전에는 끝나야하는데.. 마음이 초조합니다 :sob:

**Reply by ChaerinYu** [reply]
8점: 이번 주 PR 하나도 안..했고.. 오늘 컨디션 안 좋았는데 지금은 그냥 기분은 좋네요!ㅎㅎ

**Reply by jinsoobyeon** [reply]
5점: 크롬에서 검색을 하면 중국어로 나와요..... 언어는 분명 한글로 설정이 되있는 데요ㅠㅠ 혹시 살려 주실분 있으실까요..

**Reply by MINGDY98** [reply]
5점 : 같은 부분을 계속 공부해도 며칠 뒤면 까먹습니다 ㅠ0ㅠ

**Reply by zereight** [reply]
8점: 모기잡아서 꿀잠예약

**Comment by bigsaigon333** [comment]
체크아웃 및 회고

스터디 목표
스터디 좋았던 점
건의사항
기타 등등

- 현재의 기분을 10점 만점으로 표현해주세요~ 이유도 적어주세요!

**Reply by MINGDY98** [reply]
10점 : 공부했던 것이 복습이 되어 기억에 오래 남을것같습니다~

**Reply by ritajeong** [reply]
5점 : 이번주차 PR을 못 올려서 개인적으로 아쉽네요 ㅜ 담주에는 PR을 꼭 올려보도록 하겠습니다 도전~~

**Reply by bigsaigon333** [reply]
8점: 한번 말을 해보는게 얼마나 중요한지 다시 생각해봅니다..

100% 완벽하게 말하려고 하지 말고, 핵심 키워드를 말하려고 노력해야겠습니다!

**Reply by ChaerinYu** [reply]
7점: 당연하게 생각한 개념들을 명확하게 설명 못해서 제 자신에게 실망했습니다 ㅠㅜ 내 언어로 한 문장으로 깔끔하게 정리하는 연습 좀 많이 해봐야겠습니다..

**Reply by zereight** [reply]
9점: 까먹은 내용에 대해서 다시 생각해볼 수 있는 기회가 되어서 좋았습니다!

**Reply by jinsoobyeon** [reply]
10점: 역시 오늘도 모의 면접이 도움되었고 개선할 점도 생겼습니다!!
